### PR TITLE
Clean up Mac install instructions

### PR DESCRIPTION
**Pull request by julianoes**
There were some duplications and outdated stuff. This should clean it up
a bit.

@LorenzMeier does this all make sense?

### CODE REVIEW

**Comment by hamishwillee** [comment]
FWIW looks good to me. The only thing I'd do is add links to the docs for gazebo and jmavsim.

File: en/setup/dev_env_mac.md [hunk 1]
```diff
@@ -1,51 +1,65 @@
-# Installing Files and Code
+# Develepment Environment on Mac
```
Comment (mhkabir) [inline en/setup/dev_env_mac.md]: typo

Comment (julianoes) [reply en/setup/dev_env_mac.md]: Fixed in 716d162.

**Comment by julianoes** [comment]
@hamishwillee added in b16a793

**Comment by hamishwillee** [comment]
@mhkabir I have merge but not review rights. I am happy with this.

**Comment by mhkabir** [comment]
Done, thanks!

**Pull request merged**